validator/src/main/java/com/google/daq/mqtt/validator/SequenceValidator.java

```diff
@@ -424,8 +438,9 @@ private void queryState() {
    */
   @After
   public void tearDown() {
-    if (traceLogLevel()) {
-      warning("Not resetting config b/c trace is enabled.");
+    recordMessages = false;
+    if (debugLogLevel()) {
+      warning("Not resetting config because debug is enabled.");
```

Comment:
It isn't clear at first glance why you don't reset the config in this case. Can you add a comment to quickly spell it out?